### PR TITLE
Fix/11384 draw clippath

### PR DESCRIPTION
**Pull request by Zeljko-Predjeskovic**

### CODE REVIEW

File: Svg.Tests.Win/SvgClipPathTest.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+        var svgDoc = SvgDocument.Open<SvgDocument>("Assets\\clip_path.svg");
+
+        // Act
+        var bitMap = svgDoc.DrawDocument(backgroundColor:Color.Create(255,255,255));
+        using var file = new FileSystem().OpenWrite("Assets\\test_clip_path.png");
+        
+        bitMap.SavePng(file, 100);
+        file.Close();
+
```
Comment (gentledepp) [inline Svg.Tests.Win/SvgClipPathTest.cs:44]: Well... as your comments are indicating: There is no **Assert** part here.
So this is technically no test.

Can you please look at the other existing tests or e.g. the planrendering tests?
There we load a bitmap of what _should_ be rendered.
Then we compare the _actually rendered_ bitmap with it and create a "diff bitmap" (i.e. a bitmap with a black dot everywhere the two bitmaps do not match)
Only if they are 90% or even 100% alike, we let the test pass!

```
using var c = TestHelper.ImageCompare(svgBitmap, pngBitmap);


            c.AssertAreSimilar(92f, svgPath);
```

**Review by gentledepp — CHANGES_REQUESTED** [review]
Please complete the one test you have.
It would be even better if you could come up with some more.
But one _correct_ test would already be greatly appreciated ;-)

File: Svg.Tests.Win/SvgTextTests.cs [hunk 2]
```diff
@@ -142,7 +142,7 @@ public void CanRenderTSpan_DX_WithTransforms()
             using var c = TestHelper.ImageCompare(svgBitmap, pngBitmap);
 
 
-            c.AssertAreSimilar(99f, svgPath);
+            c.AssertAreSimilar(94f, svgPath);
```
Comment (gentledepp) [inline Svg.Tests.Win/SvgTextTests.cs]: this is worrysome.
You should not have to reduce the similarity once you implemented a feature. This indicates that you have a regression (i.e. you introduced a render bug)

Actually, when implementing stuff, the similarity should rise!

File: Svg/Platform/SkiaGraphics.cs [hunk 3]
```diff
@@ -136,6 +142,41 @@ private void SetSmoothingMode(SKPaint paint)
             }
         }
 
+        public void SetClip(GraphicsPath path, CombineMode combineMode)
+        {
+            var op = SKRegionOperation.Union;
+            switch (combineMode)
+            {
+                case CombineMode.Complement:
+                    // TODO LX is this correct?
```
Comment (eszterhorvath) [inline Svg/Platform/SkiaGraphics.cs]: @gentledepp is this correct? :D

**Review by gentledepp — APPROVED** [review]

**Pull request merged**